identity/identity.go

```diff
@@ -0,0 +1,169 @@
+package identity
```

Comment:
I think we can put this in the `config/` package. It's essentially config. Unless there is an argument against it.